tmdb-api/src/commonMain/kotlin/app/moviebase/tmdb/model/TmdbPeopleModel.kt

```diff
@@ -127,13 +127,13 @@ data class TmdbCrew(
 
 @Serializable
 data class TmdbCast(
-    @SerialName("adult") val adult: Boolean,
-    @SerialName("gender") val gender: TmdbGender,
-    @SerialName("id") override val id: Int,
+    @SerialName("adult") val adult: Boolean = false,
+    @SerialName("gender") val gender: TmdbGender = TmdbGender.UNKNOWN,
```

Comment:
I guess TMDB sends unknown if the genre isn't available. If the server doesn't deliver the value, it might be better to set `null`.